EventsExpress/ClientApp/src/components/helpers/form-helpers/render-select-field.jsx

```diff
@@ -3,22 +3,30 @@ import { makeStyles } from '@material-ui/core/styles';
 import InputLabel from '@material-ui/core/InputLabel';
 import FormControl from '@material-ui/core/FormControl';
 import Select from '@material-ui/core/Select';
-import { renderFieldError } from './';
+import { renderFieldError } from '.';
 
-const useStyles = makeStyles((theme) => ({
+const useStyles210 = makeStyles((theme) => ({
     formControl: {
         minWidth: 210,
     },
 }));
 
-export default ({ input, label, meta: { touched, error, invalid }, children }) => {
-    const classes = useStyles();
+const useStyles100 = makeStyles((theme) => ({
+    formControl: {
+        minWidth: 100,
+    },
+}));
+
+export default ({ input, label, meta: { touched, error, invalid }, children, fullWidth }) => {
+    const classes = useStyles210();
+    const classesInventories = useStyles100();
     return (
-        <FormControl variant="outlined" className={classes.formControl} >
+        <FormControl variant="outlined" className={fullWidth ? classes.formControl : classesInventories.formControl} >
```

Comment:
seems strange... let's pass minWidth from the parent, so can avoid dependency from `classesInventories` or so